### PR TITLE
Fix map and script project reference

### PR DESCRIPTION
**Pull request by hungnguyen94**
This will include a reference to Project in the Map and Script entity.

### CODE REVIEW

File: src/main/webapp/app/entities/map/map-dialog.controller.js [hunk 1]
```diff
@@ -5,11 +5,12 @@
         .module('thefirstorderApp')
         .controller('MapDialogController', MapDialogController);
 
-    MapDialogController.$inject = ['$timeout', '$scope', '$stateParams', '$uibModalInstance', 'entity', 'Map'];
+    MapDialogController.$inject = ['$timeout', '$scope', '$stateParams', '$uibModalInstance', 'entity', 'Map', 'Project'];
 
-    function MapDialogController ($timeout, $scope, $stateParams, $uibModalInstance, entity, Map) {
+    function MapDialogController ($timeout, $scope, $stateParams, $uibModalInstance, entity, Map, Project) {
```
Comment (rubenwiersma) [inline src/main/webapp/app/entities/map/map-dialog.controller.js]: We should eventually have a lower number of arguments, a task has been created in the backlog to reduce this amount

File: src/test/javascript/karma.conf.js [hunk 2]
```diff
@@ -46,7 +46,7 @@ module.exports = function (config) {
             'src/main/webapp/bower_components/blob-polyfill/Blob.js',
             'src/main/webapp/bower_components/file-saver.js/FileSaver.js',
             'src/main/webapp/bower_components/angular-file-saver/dist/angular-file-saver.bundle.js',
-            'src/main/webapp/bower_components/vis/dist/vis.js',
+            'src/main/webapp/bower_components/vis/dist/vis.min.js',
```
Comment (rubenwiersma) [inline src/test/javascript/karma.conf.js]: Good edit.

**Comment by rubenwiersma** [comment]
Looks good. We'll need to fix large argument numbers next week as it's not a pressing issue, but something SIG signalled.

File: src/main/webapp/app/entities/map/map-dialog.controller.js (hunk 1 above)
Comment (hungnguyen94) [inline src/main/webapp/app/entities/map/map-dialog.controller.js]: Maybe we can create an Angular service that combines most of the overlapping services?

**Pull request merged**